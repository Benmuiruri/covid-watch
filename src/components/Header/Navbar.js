/* eslint-disable react/button-has-type */
// @ts-nocheck
import React, { useContext } from 'react';
import AuthContext from '../../redux/auth-context';

import classes from './Navbar.module.css';

const Navbar = () => {
  const ctx = useContext(AuthContext);
  return (
    <nav className={classes.nav}>
      <ul>
        {ctx.isLoggedIn && <span>Search</span>}
        {ctx.isLoggedIn && (
          <li>
            <button onClick={ctx.onLogout}>Logout</button>
          </li>
        )}
      </ul>
    </nav>
  );
};

export default Navbar;
