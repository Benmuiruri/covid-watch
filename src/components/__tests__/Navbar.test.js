import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../Header/Navbar';

describe('Test presence of navigation header', () => {
  test('renders the navigation correctly', () => {
    const navComponent = renderer
      .create(
        <BrowserRouter>
          <Navbar />
        </BrowserRouter>,
      )
      .toJSON();
    expect(navComponent).toMatchSnapshot();
  });
});
