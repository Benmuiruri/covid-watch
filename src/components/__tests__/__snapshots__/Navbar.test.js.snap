// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test presence of navigation header renders the navigation correctly 1`] = `null`;
